File: Conditional_Statements_Advanced_Exercise/new_house.py
flower = input()
quantity = int(input())
budget = int(input())

total_sum = 0

if flower == "Roses":
    total_sum = quantity * 5
    if quantity > 80:
        total_sum -= total_sum * 0.10

if flower == "Dahlias":
    total_sum = quantity * 3.80
    if quantity > 90:
        total_sum -= total_sum * 0.15

if flower == "Tulips":
    total_sum = quantity * 2.80
    if quantity > 80:
        total_sum -= total_sum * 0.15

if flower == "Narcissus":
    total_sum = quantity * 3
    if quantity < 120:
        total_sum += total_sum * 0.15

if flower == "Gladiolus":
    total_sum = quantity * 2.50
    if quantity < 80:
        total_sum += total_sum * 0.20


if budget >= total_sum:
    total_price = budget - total_sum
    print(f"Hey, you have a great garden with {quantity} {flower} and {total_price:.2f} leva left.")
else:
    total_price = total_sum - budget
    print(f"Not enough money, you need {total_price:.2f} leva more.")

File: C++/Basics/ConditionalStatements_Advanced_Exercise/NewHouse/NewHouse.cpp
#include <iostream>
using namespace std;

int main()
{
	string flowers;
	int numFlowers, budget;

	cin >> flowers >> numFlowers >> budget;

	double totalSum = 0;

	if (flowers == "Roses") {
		totalSum = numFlowers * 5;
		if (numFlowers > 80) {
			totalSum -= totalSum * 0.10;
		}
	}
	else if (flowers == "Dahlias") {
		totalSum = numFlowers * 3.80;
		if (numFlowers > 90) {
			totalSum -= totalSum * 0.15;
		}
	}
	else if (flowers == "Tulips") {
		totalSum = numFlowers * 2.80;
		if (numFlowers > 80) {
			totalSum -= totalSum * 0.15;
		}
	}
	else if (flowers == "Narcissus") {
		totalSum = numFlowers * 3;
		if (numFlowers < 120) {
			totalSum += totalSum * 0.15;
		}
	}
	else if (flowers == "Gladiolus") {
		totalSum = numFlowers * 2.50;
		if (numFlowers < 80) {
			totalSum += totalSum * 0.20;
		}
	}

	cout.setf(ios::fixed);
	cout.precision(2);

	if (budget >= totalSum) {
		double totalPrice = budget - totalSum;

		cout << "Hey, you have a great garden with " << numFlowers << " " << flowers << " and " << totalPrice << " leva left." << endl;
	}
	else {
		double totalPrice = totalSum - budget;

		cout << "Not enough money, you need " << totalPrice << " leva more." << endl;
	}
}